index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+    include "includes/header.php";
+
+    //write PHP here...
+    $weight = '225';
+    $age = '43';
+    $first_name = 'Tony';
+    $last_name = 'Mainelli';
+
+    function poundsToKg()
+    {
+        global $weight, $Kg;
+
+        $Kg = $weight * (1/ 2.2);
+    }
+    poundsToKg();
```

Comment:
since you are using globals - you don't need to store the return of this - but again - that wasn't quite the intention of the assignment. I had hoped for:
$kg = poundsToKG($weight);